app/uk/gov/hmrc/helptosavefrontend/controllers/ReminderController.scala

```diff
@@ -234,7 +255,29 @@ class ReminderController @Inject() (
 
     }(loginContinueURL = routes.ReminderController.getRendersConfirmPage(email, period, "page").url)
 
-  def getSelectedRendersPage(): Action[AnyContent] =
+  def getSelectedRendersPage(): Action[AnyContent] = {
+    authorisedForHtsWithNINO { implicit request ⇒
+      implicit htsContext ⇒
+        helpToSaveService
+          .getAccount(htsContext.nino, UUID.randomUUID())
+          .fold(
+            e => {
+              logger.warn(s"error retrieving Account details from NS&I, error = $e")
+
+              internalServerError()
+            }, { account => {
+              if (account.isClosed) {
+                def bckLink: String = routes.ReminderController.getEmailsavingsReminders.url
+                Ok(accountClosed(Some(bckLink),account.closureDate.getOrElse(LocalDate.now())))
+              } else {
+                SeeOther(routes.ReminderController.accountOpengetSelectedRendersPage.url)
+              }
+            }
+            })
+    }(loginContinueURL = routes.ReminderController.selectRemindersSubmit.url)
+  }
+
+  def accountOpengetSelectedRendersPage(): Action[AnyContent] =
```

Comment:
Camel case: that should read accountOpenGetSelectedRendersPage

Reply:
Sweet Ill amend that now!

Reply:
Mended those changes